vowpalwabbit/global_data.h

```diff
@@ -537,7 +532,9 @@ struct vw
   time_t init_time;
 
   std::string final_regressor_name;
-  regressor reg;
+
+  uint32_t stride_shift;
```

Comment:
It won't take too long but we'll end up with something like - 
```new(&all.weights) weight_parameters(length, all.weights.stride_shift()); ``` in parse_regressor to get the stride_shift. Is that okay?